pkg/store/image/image.go

```diff
@@ -40,8 +40,9 @@ type Image struct {
 	ChainID string
 	// Size is the compressed size of the image.
 	Size int64
-	// Config is the oci image config of the image.
-	Config *imagespec.ImageConfig
+	// ImageSpec is the oci image structure which describes basic information about the image.
+	ImageSpec imagespec.Image
+
```

Comment:
Unnecessary empty line. I'll clean this up in my coming cleanup PR.

Reply:
ok....?

Reply:
Done.